Makefile

```
@@ -8,13 +8,13 @@ install-hatch:
 
 # This edits your local pre-commit hook file to use Hatch when executing.
 overwrite-pre-commit:
-	hatch run pre-commit:pre-commit install
-	hatch run pre-commit:sed -i -e "s/exec /exec hatch run pre-commit:/g" .git/hooks/pre-commit
+	hatch run dev-env:pre-commit install
+	hatch run dev-env:sed -i -e "s/exec /exec hatch run pre-commit:/g" .git/hooks/pre-commit
```

I pulled down the branch and made an empty commit just to do a sanity check. Doing so gave me the error
`Unknown environment: pre-commit`. It looks like the follow change needs to be made so that hatch runs pre-commit properly.
```suggestion
	hatch run dev-env:sed -i -e "s/exec /exec hatch run dev-env:/g" .git/hooks/pre-commit
```

---

Makes sense - thanks!